### PR TITLE
Change game post

### PR DESCRIPTION
**Pull request by Safisyx**
Basically! We just don't want to send the board2 when creating a new game, although it is not really necessary as board2 is still empty on that time :sweat_smile: , we really need that in the updates but maybe just for safety now haha

### CODE REVIEW

File: src/games/entities.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import { BaseEntity, PrimaryGeneratedColumn, Column, Entity, Index, OneToMany, ManyToOne } from 'typeorm'
 import User from '../users/entity'
 
-export type Row = [ string , string ,string ,string ,string ,string ,string ,string ,string ,string]
-export type Board = [ Row, Row, Row, Row, Row, Row,Row, Row, Row, Row]
```
Comment (Ju1111) [inline src/games/entities.ts:5]: thanks for removing this! It's prettier now 😆

File: src/games/entities.ts [hunk 2]
```diff
@@ -32,9 +33,20 @@ export class Game extends BaseEntity {
...
 
+  @Column('boolean', {default: false})
+  p1ready: boolean
+
+  @Column('boolean', {default: false})
+  p2ready: boolean
+
+  @Column('char', {length:1, default: '1'})
+  turn: Symbol
+
```
Comment (Ju1111) [inline src/games/entities.ts:44]: we need to keep an eye on this, if it actually works how we think or not

File: src/games/entities.ts [hunk 3]
```diff
@@ -32,9 +33,20 @@ export class Game extends BaseEntity {
...
   @Column('text', {default: 'pending'})
   status: Status
 
+
```
Comment (Safisyx) [inline src/games/entities.ts:48]: :blowfish:

**Review by Ju1111 — APPROVED** [review]

File: src/games/entities.ts (hunk 1 above)
Comment (Safisyx) [reply src/games/entities.ts:5]: Sure :+1:

**Pull request merged**